SpotifySearch/SpotifySearch/Album.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Album.swift
+//  SpotifySearch
+//
+//  Created by Eric Chang on 10/28/16.
+//  Copyright © 2016 Eric Chang. All rights reserved.
+//
+
+import Foundation
+
+struct Album {
```

Comment:
Why did you choose to make Album a struct and not a class? I'm not saying it's wrong just curious.